langstream-cli/src/main/java/ai/langstream/cli/commands/applications/UIAppCmd.java

```diff
@@ -30,7 +30,10 @@
 import io.undertow.server.handlers.resource.ResourceHandler;
 import io.undertow.util.Headers;
 import io.undertow.util.HttpString;
+import java.awt.Desktop;
```

Comment:
this is a great idea !

I think that Desktop.browse is meant to be compatible with all the OSs
https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/awt/Desktop.html#browse(java.net.URI)

I have tested it on Mac and it works like a charm

Can you please try to use it also on your OS ?

Reply:
It doesn't work in WSL linux.  I'll modify the code to launch if in a mac. For WSL linux, I tried it earlier. The WSL error was something like "can't launch headless".  Easy to try though.  Will try again.

Reply:
Tried this.  At least I got a status message in the logs:
```
received response: (GET http://localhost:8090/api/applications/default/test?stats=false) 200
Starting UI at http://localhost:8092
Operating system identified as: linux
Could not Start browser.  Either add the proper command to your OS (open for mac or xdg-open for linux) or start a browser manually at http://localhost:8092
```

code change:
```

    static boolean openBrowserAtPort(String URL, int port) {
        String os = System.getProperty("os.name").toLowerCase();
        String openCommand = "";
        if (os.indexOf("win") >= 0
                || os.indexOf("nux") >= 0
                || os.indexOf("nix") >= 0
                || os.indexOf("nux") >= 0) {
            Desktop desktop = Desktop.getDesktop();
            try {
                desktop.browse(new URI(URL + port));
            } catch (Exception e) {
                return false;
            }
            return true;
        } else if (os.indexOf("nix") >= 0 || os.indexOf("nux") >= 0) {
            openCommand = "/usr/bin/xdg-open";
            return checkAndLaunch(openCommand, port);
        }
        return false;
```
    }

Reply:
I will, however, have desktop browser to open the browser on a mac